src/main/webapp/styles.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["styles.scss","styles.css"],"names":[],"mappings":"AAAA;EACI,iBAAA;EACA,kBAAA;EACA,YAAA;ACCJ;;ADEE;EACE,gBAAA;ACCJ","file":"styles.css"}
```

Comment:
Hi @erivas28 if you don't mind me asking what do the styles.css.map file do?

Reply:
I believe it is automatically created file, because I never manually created that :)